### DISCUSSION TITLE
Rendering PDF from bundled worker vs CDN yields different results

### DISCUSSION THREAD

**Discussion by hngu** [wojtekmaj/react-pdf#951] [Q&A] [answered]
Hi,

I was going to post a bug but I am not sure if I misconfigured something. When I render a PDF (which I cannot share due to sensitive data) using the bundled worker like this:
```
import { Document, Page } from 'react-pdf/dist/esm/entry.webpack';
```
The PDF renders correctly. The demo react-pdf posted in the home page also renders the PDF correctly as well.

However, if I want to use the pdf worker from CDN and configure like so:
```
import {Document, pdfjs} from 'react-pdf;
pdfjs.GlobalWorkerOptions.workerSrc = `//cdnjs.cloudflare.com/ajax/libs/pdf.js/${pdfjs.version}/pdf.worker.js`;
```
The PDF DOES NOT RENDER. Looking at the build for esm entry, it looks like it is using the `/legacy` version of `pdfjs-dist`. Is that intentional? Also, is the configuration I have correct? Thanks!

**Comment by wojtekmaj** [comment]
It's intentional. 

PDF.js has introduced "modern" and "legacy" builds (previously, "legacy" was the only build available).

We're using "legacy" build for the sake of supporting IE 11. Since v5.4.0, documentation was updated and suggested `workerSrc` has been changed. You should definitely keep it in line with the latest recommendations!

**Comment by hngu** [comment, marked as answer]
Closing - looks like for my issue I had to get the right fonts loaded:
```
            options={{
              cMapUrl: `//cdn.jsdelivr.net/npm/pdfjs-dist@${pdfjs.version}/cmaps/`,
              cMapPacked: true,
              standardFontDataUrl: `https://unpkg.com/browse/pdfjs-dist@${pdfjs.version}/standard_fonts/`,
            }}
```
